coderd/pubsub/inboxnotification.go

```diff
@@ -0,0 +1,44 @@
+package pubsub
```

Comment:
Howcome you're creating this package inside `coderd` and not `coderd/notifications`?

Reply:
I may have know the answer. It looks awkward but it seems ~consistent with `coderd/wspubsub`. You have 2 options:
1. Rename `wspubsub` to `pubsub`, and park everything there.
2. Just add leave this code in `coderd/notifications`